openapi-specs/schema.json

```diff
@@ -3984,7 +3984,15 @@
             "in": "path",
             "required": true,
             "type": "string"
-          }
+          },
+          {
+            "name": "consistency",
+            "in": "header",
+            "required": false,
+            "default": false,
```

Comment:
I believe the default should be true for the HTTP clients. since consistency=false would be used just for debugging only